### ISSUE TITLE
library "libz.so.1" not found

### ISSUE THREAD

**Issue by uomar8** [itsaky/AndroidIDE#7]
After first build it says 

```
========== [15:47:44] TASK_BEGIN: Start Completion Services ==========
CANNOT LINK EXECUTABLE "ANDROIDIDE_HOME/jdk/bin/java": library "libz.so.1" not found
CANNOT LINK EXECUTABLE "ANDROIDIDE_HOME/jdk/bin/java": library "libz.so.1" not found
```

Information:
Device: Galaxy Tab s3
Android 9
app version: 1.0.2.1-alpha
Jdk: 07.08.2021

**Comment by itsaky** [comment]
This is known issue and will be fixed in next update.

However, a temporary fix is that you wait for some time before opening a project.

**Comment by uomar8** [comment]
> This is known issue and will be fixed in next update.
> 
> However, a temporary fix is that you wait for some time before opening a project.
> 
> Let me know if this doesn't work.

I tried to wait and it didn't work, also tried to create new project and wait also didn't work

**Comment by itsaky** [comment]
Hey @uomar8,

We're working to fix this issue. However, I can't reproduce the issue on my device. Could you test the newly built JDK to check if this is really fixed or not?

**Comment by uomar8** [comment]
@itsaky I just tried not fixed yet 

I founded another away to fix this problem but it make another problem 

What I did 

Copy AndroidIDE.apk/assets/data/lib/libz.so.1 to jdk.tar.xz/lib/
And the new problem is 

Error: dl failure on line 598
Error: failed ANDROIDIDE_HOME/jdk/lib/server/libjvm.so, because dlopen failed: library "libc++_shared.so" not found

So my conclusion is the app won't copy /assets/data/lib/ to /usr/lib/
I don't know how to do this manually

**Comment by itsaky** [comment]
Dude, I just recompiled JDK with a fix for that. Could you help me test it?

You just have to download the new JDK and reinstall AndroidIDE.

**Comment by uomar8** [comment]
@itsaky 

I just tried to use the new jdk and it still says 

========== [14:03:50] TASK_BEGIN: Start Completion Services ==========
CANNOT LINK EXECUTABLE "ANDROIDIDE_HOME/jdk/bin/java": library "libz.so.1" not found
CANNOT LINK EXECUTABLE "ANDROIDIDE_HOME/jdk/bin/java": library "libz.so.1" not found

**Comment by itsaky** [comment]
The new JDK is not available publicly at the moment. This is because it hasn't been tested yet. What JDK did you tested?

Test version of JDK is available [here](https://drive.google.com/file/d/1ODwuMY58i71sPEssVl-_lVMgoIKHicRP/view?usp=drivesdk).

**Comment by uomar8** [comment]
@itsaky 

I just tried the test version  there is a new error 

Gradle Daemons Stopped
cd "/storage/emulated/0/AndroidIDEProjects/test/app"


========== [14:38:20] TASK_BEGIN: Start Completion Services ==========
OpenJDK 64-Bit Server VM warning: No monotonic clock was available - timed services may be adversely affected if the time-of-day clock changes
Segmentation fault
OpenJDK 64-Bit Server VM warning: No monotonic clock was available - timed services may be adversely affected if the time-of-day clock changes
Segmentation fault

**Comment by karelantonio** [comment]
I think the problem is: as openjdk was not installed via a Package Manager, could have some dependencies lost (Maybe arent the same bcz are from OpenJdk 17): try to check if all of this are available: `freetype, libandroid-shmem, libandroid-spawn, libiconv, zlib, xorgproto, libx11, libxcursor, libxext, cups, fontconfig, libpng, libxrender, libxtst, libxrandr, libxt, libxi`

**Comment by itsaky** [comment]
Fixed in v1.0.3-alpha.

**Comment by uomar8** [comment]
BUILD SUCCESSFUL in 1m 21s
Yes it fixed in 1.0.3 thank you
